scripts/load-move-book/main.py

```diff
@@ -77,26 +80,17 @@ def store_in_db_via_api(table_name, paragraphs, embeddings):
 
 # Main function to handle the entire flow
 def process_pdf_to_zerokdb(pdf_path, table_name="pdf_data", create_table=True):
-    # Extract text
     text = extract_text_from_pdf(pdf_path)
-
-    # Optionally create a new table
     if create_table:
         create_table_if_not_exists(table_name)
-
-    # Split the text into paragraph chunks
     paragraphs = split_text_into_paragraphs(text)
-
-    # Generate embeddings for each paragraph
     embeddings = [generate_embeddings_via_api(paragraph) for paragraph in paragraphs]
-
-    # Store data in ZeroKDB via API
     store_in_db_via_api(table_name, paragraphs, embeddings)
 
 # Entry point
 if __name__ == "__main__":
     pdf_path = "data/move-book.pdf"
-    table_name = f"move_book"
+    table_name = "move_book_test_3"
```

Comment:
I think we should change this to be the_move_book (I will update that on the frontend env variable to match)

Also if you could populate the new table already with all the move book data